examples.sh

```diff
@@ -0,0 +1,190 @@
+                    </front>
+                </content>
+            </list-item>
+            <list-item>
+                <language>en</language>
+                <name>body</name>
+                <content>
+                    <body xmlns="http://libero.pub" xml:lang="en">
+                        <p>This is the body of the article.</p>
+                    </body>
+                </content>
+            </list-item>
+            <list-item>
+                <language>es</language>
+                <name>front</name>
+                <content>
+                    <front xmlns="http://libero.pub" xml:lang="es">
+                        <id>1234</id>
+                        <title>Este es el título del artículo</title>
+                    </front>
+                </content>
+            </list-item>
+        </content-list>
+    </root>'
+
+line
+line
+echo "Take a look at the Journal at http://localhost:8080, you'll see the article."
+echo "Also take a look at the dashboard at http://localhost:8082, you'll see the history of the run."
+hold
+
+#
+# Step 2: create article version 2
+#
+
+echo "Next we'll create version 2 of article 1234"
+hold
+
+curl --verbose http://localhost:8085/articles/1234/versions -X POST --header 'Content-Type: application/xml' --data '
+    <root>
+        <content-list>
+            <list-item>
+                <language>en</language>
+                <name>front</name>
+                <content>
+                    <front xmlns="http://libero.pub" xml:lang="en">
+                        <id>1234</id>
+                        <title>This is the title of the article (version 2)</title>
+                    </front>
+                </content>
+            </list-item>
+            <list-item>
+                <language>en</language>
+                <name>body</name>
+                <content>
+                    <body xmlns="http://libero.pub" xml:lang="en">
+                        <p>This is the body of the article (version 2).</p>
+                    </body>
+                </content>
+            </list-item>
+            <list-item>
+                <language>es</language>
+                <name>front</name>
+                <content>
+                    <front xmlns="http://libero.pub" xml:lang="es">
+                        <id>1234</id>
+                        <title>Este es el título del artículo (versión 2)</title>
+                    </front>
+                </content>
+            </list-item>
+        </content-list>
+    </root>'
+
+line
+line
+echo "Take a look at the Journal at http://localhost:8080, you'll see version 2 of the article."
+echo "Also take a look at the dashboard at http://localhost:8082, you'll see the history of both runs."
+hold
+
+#
+# Step 3: update article version 2
+#
+
+echo "Now we'll update version 2 of article 1234"
+hold
+
+curl --verbose http://localhost:8085/articles/1234/versions/2 -X PUT --header 'Content-Type: application/xml' --data '
+    <root>
```

Comment:
this XML content can be stored in a separate file and loaded with `--data @filename.xml`